### PR TITLE
feat: add loading component

### PR DESCRIPTION
**Pull request by ManuViola77**
## Links:
[Agregar un Loader lindo](https://github.com/users/ManuViola77/projects/1/views/1#:~:text=Agregar%20un%20Loader%20lindo)

## What & Why:
This PR adds the Loading Component

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

**Review by jpirazusta — APPROVED** [review]

**Comment by fernandorosenblit** [comment]
Good job! But why don't we create a funny SVG animation? As an inspiration, we can animate "wordle" like this example https://www.behance.net/gallery/61343193/Open-source-Animated-Words-Pack-01/modules/360640345

**Comment by ManuViola77** [comment]
> Good job! But why don't we create a funny SVG animation? As an inspiration, we can animate "wordle" like this example [behance.net/gallery/61343193/Open-source-Animated-Words-Pack-01/modules/360640345](https://www.behance.net/gallery/61343193/Open-source-Animated-Words-Pack-01/modules/360640345)

ooooh niiiice!!! love the idea! I didn't know how to do that. I will investigate and try it out, thanks!!!

File: src/components/common/Loading/index.js [hunk 1]
```diff
@@ -1,11 +1,102 @@
...
+
+const Loading = () => {
+  const styles = {
+    container: { height: '50px', width: '50px' },
+    correct: { backgroundColor: LETTER_STATUS.correct },
+    incorrect: { backgroundColor: LETTER_STATUS.incorrect },
+    misplaced: { backgroundColor: LETTER_STATUS.misplaced },
+  };
+
+  const loadingWords = [
```
Comment (jpirazusta) [inline src/components/common/Loading/index.js]: I think we could move these 2 const outside of the component

File: src/components/common/Loading/index.js [hunk 2]
```diff
@@ -1,11 +1,102 @@
...
 
-const Loading = () => (
-  <div className="loading-container">
-    <img src={loading} className="loading-icon" alt="loading" />
-  </div>
-);
+  return (
+    <div className="loading-container">
+      {loadingWords.map(({ front, back, frontStyle, backStyle }, index) => (
+        <ReactFlipCard
```
Comment (jpirazusta) [inline src/components/common/Loading/index.js:90]: I think we should add a `key` here

**Review by jpirazusta — APPROVED** [review]
😍 😍

**Review by fernandorosenblit — APPROVED** [review]

**Pull request merged**